src/main/java/com/tobmistaketracker/detector/VerzikP3MistakeDetector.java

```diff
@@ -111,6 +112,26 @@ public void onGraphicChanged(GraphicChanged event) {
         }
     }
 
+    @Subscribe
+    public void onGameTick(GameTick tick)
+    {
+        NPC verzik = client.getNpcs().stream()
+                .filter(VerzikP3MistakeDetector::isVerzikP3)
+                .findFirst()
+                .orElse(null);
+
+        verzikMeleeChancedTracker.setVerzikAttackInfo(verzik);
+    }
+
+    @Subscribe
+    public void onAnimationChanged(AnimationChanged event)
+    {
+        if (event.getActor() instanceof NPC)
+        {
+            verzikMeleeChancedTracker.checkPlayerWronglyTanked(event);
+        }
```

Comment:
We could probably keep the logic all in this class if we really wanted. Not a huge deal though, but I think it would be easier to just read if we added the logic to this class.